examples/next-notes/package.json

```diff
@@ -10,7 +10,7 @@
     "lint": "next lint"
   },
   "dependencies": {
-    "@ceramicnetwork/http-client": "^2.2.1",
+    "@ceramicnetwork/http-client": "^2.3.0-rc.0",
```

Comment:
Can you use the stable versions instead of the RCs now that they have been released please?